parsers/JP.py

```diff
@@ -210,9 +215,14 @@ def fetch_consumption_forecast(
     # Currently past dates not implemented for areas with no date in their demand csv files
     if target_datetime and zone_key == "JP-HKD":
         raise NotImplementedError("Past dates not yet implemented for selected region")
-    datestamp = arrow.get(target_datetime).to("Asia/Tokyo").strftime("%Y%m%d")
+
+    if target_datetime is None:
```

Comment:
When `target_datetime == None`, `arrow.get(target_datetime)` returns the current time.